superset-frontend/src/views/CRUD/data/savedquery/SavedQueryList.tsx

```diff
@@ -97,6 +98,7 @@ function SavedQueryList({
 
   const canEdit = hasPerm('can_write');
   const canDelete = hasPerm('can_write');
+  const canExport = hasPerm('can_read');
```

Comment:
We should only show the export buttons if the `VERSIONED_EXPORT` feature flag is enabled, so this needs to be something like this:

```js
const canExport =
    hasPerm('can_read') && isFeatureEnabled(FeatureFlag.VERSIONED_EXPORT);
```